desktop/core/src/desktop/middleware_test.py

```diff
@@ -156,3 +156,21 @@ def test_ensure_safe_redirect_middleware():
     settings.MIDDLEWARE.pop()
     for finish in done:
       finish()
+
+def test_spnego_middleware():
+  done = []
+  # settings.MIDDLEWARE.append('desktop.middleware.SpnegoMiddleware')
```

Comment:
nit

Reply:
fixed. Added a comment that "use SpnegoDjangoBackend to enable 'desktop.middleware.SpnegoMiddleware'"